Hypnotic-Herons/Sprint-2
# JS class name changes

# Steps to Test
Outline the steps to test

```
git fetch --all
git checkout js_classNameChange
code .
```

python manage.py makemigrations HRIT
python manage.py migrate
python manage.py createsuperuser (this is just for your local use)
login on the admin view http://127.0.0.1:8000/admin
add a training programs
add a few departments
look at the browser view for training programs & departments
you should see the training programs & departments you just added
